src/ldp/agent/simple_agent.py

```diff
@@ -19,6 +21,10 @@ class HiddenEnvStateMessage(EnvStateMessage):
     content: str = "[Previous environment state - hidden]"
 
 
+class HiddenTransitionsMessage(Message):
```

Comment:
Do you mind using `Message.info` over introducing more classes

Reply:
I prefer the classes - what's wrong with using types? We use typing to distinguish between tool request messages, tool response messages, env state messages, etc.

Reply:
as discussed in person, using  a `Message` here